### ISSUE TITLE
False positive: bugmenot.com

### ISSUE THREAD

**Issue by CactiChameleon9** [lightswitch05/hosts#208]
False positive:
bugmenot.com is blocked with your ads and tracking host. It, however, is not a tracking domain. It provides logins for people who don't want to create accounts for websites.

**Comment by lightswitch05** [comment]
Hey @CactiChameleon9 - thanks for the report. I just checked, I do not have `bugmenot.com` blocked in my list. I do however have `bugmenot.org` blocked.

`bugmenot.org` was added from this list: `https://awakesecurity.com/wp-content/uploads/2020/06/GalComm-Registered-Domains-List-Appendix-A.txt`, which is a list of malware domains discovered in [The Internet’s New Arms Dealers: Malicious Domain Registrars](https://awakesecurity.com/blog/the-internets-new-arms-dealers-malicious-domain-registrars/)

Since you mistook `bugmenot.org` for `bugmenot.com`, I'm going to assume this ticket is invalid for now.
